include/tvm/relay/expr.h

```diff
@@ -378,6 +376,23 @@ class IfNode : public ExprNode {
 
 RELAY_DEFINE_NODE_REF(If, IfNode, Expr);
 
+/*! \brief Get a field out of a tuple. */
+class GetItem;
+class GetItemNode : public ExprNode {
+ public:
+  /*! \brief The tuple */
+  Expr tuple;
+  /*! \brief which value to get */
+  size_t field;
```

Comment:
change to ```int index;```

Reply:
the main reason is that usually if we want to serialize the model, the type should have fixed size across platform(int32, or int64). index is a more commonly used term